### DISCUSSION TITLE
Updating the version from 1.7.8.5 to 1.7.8.10 - need a specialist.

### DISCUSSION THREAD

**Discussion by AlonaBau** [PrestaShop/PrestaShop#34591] [Q&A]
I need help updating the version from 1.7.8.5 to 1.7.8.10. I tried to update via module 1 Click upgrade - there was a problem with the database backup, and the update froze. I can’t cope with this on my own; I need the help of a specialist.
**Details**
Prestashop 1.7.8.5
PHP version 7.4.
The screenshot shows where the backup freezes
![back](https://github.com/PrestaShop/PrestaShop/assets/151015341/b60a96b4-1707-4fa4-8a44-5a5ba6992505)

**Comment by MatShir** [comment]
You can find help [here](https://www.prestashop-project.org/support/). If the issue has not been faced by other members of the community.
